enclave-runner/src/usercalls/mod.rs

```diff
@@ -1285,7 +1287,6 @@ impl<'tcs> IOHandlerInput<'tcs> {
 
     #[inline(always)]
     fn exit(&mut self, panic: bool) -> EnclaveAbort<bool> {
-        self.enclave.abort_all_threads();
```

Comment:
I don't think we should remove this. We should avoid scheduling other threads ASAP after exit is called. Instead, can we have special logic in `syscall_loop` that waits for an exit that isn't `EnclaveAbort::Secondary`?

Reply:
That's cleaner, I updated with the PR.